.github/dependabot.yml

```diff
@@ -11,3 +11,8 @@ updates:
       # ignore all patch updates
       - dependency-name: '*'
         update-types: ['version-update:semver-patch']
+
+      # @date-io/luxon@1 requires luxon@1
+      - dependency-name: 'luxon'
+      # luxon@1 requires @types/luxon@1
+      - dependency-name: '@types/luxon'
```

Comment:
Merge issue? I see there are Makefile changes too

Reply:
Ah I thought I fixed it with the revert. Yeah

Reply:
Not sure why it's all showing up since I merged `message-logs-frontend` into this branch. I reverted the accidental master merge

Reply:
Should be fixed now. I was trying to get the changes from #2088 into this so it could pass on CI